supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesAsyncBackendJobExecutionActor.scala

```diff
@@ -59,6 +61,119 @@ case object Cancelled extends TesRunStatus {
 
 object TesAsyncBackendJobExecutionActor {
   val JobIdKey = "tes_job_id"
+  private def generateLocalizedSasScriptPreamble(environmentVariableName: String, getSasWsmEndpoint: String) : String = {
```

Comment:
Do you think there's any value in a test of this method?

Reply:
Yeah. I added a unit test to confirm that the `curl` and `export` commands are generated and included as we would expect. 

Still thinking on how to actually run this code in an integration test setting.

Reply:
Since it requires WSM integration I don't think we need to include integration testing here, maybe a good case to consider in upcoming test infrastructure planning.